csfieldguide/templates/interactives/sorting-algorithms.html

```diff
@@ -0,0 +1,197 @@
+            <div id="box-u" class="ordered-box-content">
+              <img id="box-u-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+        <div class="sorting-algorithms-interactive-item">
+            <div id="box-v" class="ordered-box-content">
+              <img id="box-v-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+        <div class="sorting-algorithms-interactive-item">
+            <div id="box-w" class="ordered-box-content">
+              <img id="box-w-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+        <div class="sorting-algorithms-interactive-item">
+            <div id="box-x" class="ordered-box-content">
+              <img id="box-x-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+        <div class="sorting-algorithms-interactive-item">
+            <div id="box-y" class="ordered-box-content">
+              <img id="box-y-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+        <div class="sorting-algorithms-interactive-item">
+            <div id="box-z" class="ordered-box-content">
+              <img id="box-z-img" class="placeholder-image" data-weight="0" src="{% static 'interactives/sorting-algorithms/img/white-placeholder.png' %}">
+            </div>
+        </div>
+      </div>
+
+      <div id="sorting-algorithms-order-reminder">
+        <div class="sorting-algorithms-order-reminder-text">
+          <p>{% trans "Lightest" %}</p>
+        </div>
+        <div class="sorting-algorithms-order-reminder-text">
+          <p>{% trans "Heaviest" %}</p>
+        </div>
+      </div>
+
+      <div id="sorting-algorithms-result">
+        <h3 id="check-order-result-text-incorrect">{% trans "The boxes are not in order!" %}</h3>
+        <h3 id="check-order-result-text-correct">{% trans "The boxes are in order!" %}</h3>
+        <button type="button" onclick="checkOrder()" id="check-sorted-button">{% trans "Check Order" %}</button>
+        <div id="comparision-counter">
+          <p id="comparison-counter-prefix">{% trans "Number Of Comparisons:" %}</p><p id="comparison-counter-number">0</p>
+        </div>
+      </div>
+      
+    </div> 
+
+  </div>
+{% endblock html %}
+
+{% block css %}
+  <link rel="stylesheet" href="{% static 'interactives/sorting-algorithms/css/sorting-algorithms.css' %}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dragula/3.7.2/dragula.min.css">
```

Comment:
Shouldn't use CDN, rather load `dragula` from NPM.

Reply:
agreed, but can't actually work out the process for this. Is my interactive meant to have a package.json file? Is that it?

Reply:
Yep, you will also need to list the new `package.json` in the top level `package.json`.